src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs

```diff
@@ -30,9 +31,9 @@ public class HomeWorkspaceModel : WorkspaceModel
 
         private string thumbnail;
         private Uri graphDocumentationURL;
-        private readonly DynamoScheduler scheduler;
+        private DynamoScheduler scheduler;
```

Comment:
why are these changed?

Reply:
I feel is less error prone the way we have it now. @mjkkirschner ?